### ISSUE TITLE
Add option to construct a new animation but reversed, based on an existing one

### ISSUE THREAD

**Issue by luanpotter** [luanpotter/flame#65]
Something like

```dart
  Animation animation = ...
  Animation reversed = animation.reversed(); // creates a new animation with the same frame but in reversed order
```

possible implementation:

```dart
    return new Animation(frames.reversed.toList())
```

Something like this. Would be really useful.

**Comment by luanpotter** [comment]
It'd be a good opportunity to actually fix this: the loop optional parameter in the animation nameless constructor should have default value true.

**Comment by luanpotter** [comment]
This was fixed by #66.
